share/src/main/java/io/zensoft/share/aspect/log/PostListenerLoggingAdvice.java

```diff
@@ -0,0 +1,81 @@
+package io.zensoft.share.aspect.log;
+
+import org.aspectj.lang.JoinPoint;
+import org.aspectj.lang.annotation.After;
+import org.aspectj.lang.annotation.Aspect;
+import org.aspectj.lang.annotation.Before;
+import org.aspectj.lang.reflect.MethodSignature;
+import org.slf4j.Logger;
+import org.springframework.amqp.rabbit.annotation.RabbitListener;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.stereotype.Component;
+
+import java.lang.reflect.Method;
+
+/**
+ * Created by temirlan on 6/28/18.
+ * this class listens for the execution of PostListener interface methods
+ * if execution occurs it logs with corresponding actions and descriptions
+ *
+ * @see io.zensoft.share.listener.PostListener
+ */
+@Aspect
+@Component
+public class PostListenerLoggingAdvice {
+
+
+    @Autowired
+    @Qualifier("rabbitMqMessageListenerLogger")
+    private Logger logger;
+
+    /**
+     * logs before message handled by controllers(listeners)
+     * logs :
+     * queue name where message came
+     * body of a message
+     * message handler class detail(class name,method name etc.)
+     *
+     * @param joinPoint object to extract information about message handler class
+     */
+    @Before("io.zensoft.share.aspect.PostListenerPointCut.onPublishMethodCalled()")
+    public void logBeforePublishMessageHandled(JoinPoint joinPoint) {
+        String queueNameWhereMessageCame = getQueueNameAnnotatedWithRabbitListener(joinPoint);
+        logger.info("Receiving message to queue : " + queueNameWhereMessageCame);
```

Comment:
May be it's better to rename variable queueNameWhereMessageCame to requestReceiverQueue